packages/boot/test/bootstrapTests/test-liquidation-1.ts

```diff
@@ -139,105 +143,20 @@ const checkFlow1 = async (
   const {
     advanceTimeBy,
     advanceTimeTo,
-    agoricNamesRemotes,
     check,
-    setupStartingState,
     priceFeedDrivers,
     readLatest,
     walletFactoryDriver,
+    setupVaults,
+    placeBid,
   } = t.context;
 
   const metricsPath = `published.vaultFactory.managers.manager${managerIndex}.metrics`;
 
-  await setupStartingState({
-    collateralBrandKey,
-    managerIndex,
-  });
-
-  const minter = await walletFactoryDriver.provideSmartWallet('agoric1minter');
-
-  for (let i = 0; i < setup.vaults.length; i += 1) {
-    const offerId = `open-${collateralBrandKey}-vault${i}`;
-    await minter.executeOfferMaker(Offers.vaults.OpenVault, {
-      offerId,
-      collateralBrandKey,
-      wantMinted: setup.vaults[i].ist,
-      giveCollateral: setup.vaults[i].atom,
-    });
-    t.like(minter.getLatestUpdateRecord(), {
-      updated: 'offerStatus',
-      status: { id: offerId, numWantsSatisfied: 1 },
-    });
-  }
-
-  // Verify starting balances
-  for (let i = 0; i < setup.vaults.length; i += 1) {
-    check.vaultNotification(managerIndex, i, {
-      debtSnapshot: { debt: { value: scale6(setup.vaults[i].debt) } },
-      locked: { value: scale6(setup.vaults[i].atom) },
-      vaultState: 'active',
-    });
-  }
+  await setupVaults(collateralBrandKey, managerIndex, setup);
 
   const buyer = await walletFactoryDriver.provideSmartWallet('agoric1buyer');
-  {
```

Comment:
consider keeping this block so reduce churn on whitespace (which makes it look like lines changed that didn't really)

Reply:
sorry, yeah a different one. I put the comment there now